packages/core/package.json

```diff
@@ -55,6 +56,7 @@
     "@types/lodash": "^4.14.182",
     "eslint": "^7.32.0",
     "jest": "^28.1.3",
+    "madge": "^5.0.1",
```

Comment:
Out of curiosity, couldn't this have been a devDependency at the root level, since it's needed by all projects?

Reply:
It must be defined locally if addressing the binary in `scripts`